contracts/lib/arbitration/IAragonAppFeesCashier.sol

```diff
@@ -3,16 +3,76 @@ pragma solidity ^0.4.24;
 import "../token/ERC20.sol";
 
 
-// Note
+/**
+* @title AragonAppFeesCashier interface
```

Comment:
Let's make sure we move this inline doc to aragon-court now that it's removed here :)

Reply:
Done!